scripts/pylib/twister/twisterlib/harness.py

```diff
@@ -234,16 +233,15 @@ def configure(self, instance):
     def pytest_run(self, timeout):
         try:
             cmd = self.generate_command()
-            if not cmd:
```

Comment:
why remove this check?

Reply:
in current implementation `cmd` will never be empty. If something went wrong in `self.generate_command()`, then `PytestHarnessException` will be raised and handled in the current method